notifications/src/main/kotlin/notifications/smtp/SMTPService.kt

```diff
@@ -0,0 +1,64 @@
+package notifications.smtp
+
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.map
+import mu.KLogging
+import notifications.config.SMTPConfig
+import java.util.*
+import javax.mail.*
+import javax.mail.internet.InternetAddress
+import javax.mail.internet.MimeBodyPart
+import javax.mail.internet.MimeMessage
+import javax.mail.internet.MimeMultipart
+
+/**
+ * Service that is used to send email messages using SMTP
+ */
+class SMTPService(private val smtpConfig: SMTPConfig) {
+
+    private val properties = Properties()
+
+    init {
+        properties["mail.smtp.host"] = smtpConfig.host
+        properties["mail.smtp.port"] = smtpConfig.port
+        properties.put("mail.smtp.auth", true)
+    }
+
+    /**
+     * Sends email message
+     * @param from - email of sender
+     * @param to - destination email
+     * @param subject - subject of email message
+     * @param message - message itself
+     * @return result of operation
+     */
+    fun sendMessage(from: String, to: String, subject: String, message: String): Result<Unit, Exception> {
+        return Result.of {
+            Session.getInstance(properties, object : Authenticator() {
+                override fun getPasswordAuthentication(): PasswordAuthentication {
+                    return PasswordAuthentication(smtpConfig.userName, smtpConfig.password)
+                }
+            })
+        }.map { session ->
+            val mimeMessage = MimeMessage(session)
+            mimeMessage.setFrom(InternetAddress(from))
+            mimeMessage.setRecipients(
+                Message.RecipientType.TO, InternetAddress.parse(to)
+            )
+            mimeMessage.subject = subject
+            val mimeBodyPart = MimeBodyPart()
+            mimeBodyPart.setContent(message, "text/html")
```

Comment:
I know this is the initial version but for future I suggest to use email template, for example, [FreeMarker](https://freemarker.apache.org) (flexible CSS styles support)